base/server/src/main/java/com/netscape/cmscore/authentication/ChallengePhraseAuthentication.java

```diff
@@ -378,7 +369,7 @@ private String hashPassword(String pwd) {
         String salt = "lala123";
         byte[] pwdDigest = mSHADigest.digest((salt + pwd).getBytes());
         String b64E = Utils.base64encode(pwdDigest, true);
-
-        return "{SHA}" + b64E;
+        logger.debug("Password hashed with {}", mSHADigest.getAlgorithm());
+        return String.format("{%s}%s", mSHADigest.getAlgorithm().toUpperCase(), b64E);
```

Comment:
how about the hashPassword() defind in CRSEnrollment.java, CMSServlet.java

Reply:
Accounted for in the latest patchset.